libs/libc/machine/xtensa/arch_setjmp.S_bk

```diff
@@ -0,0 +1,374 @@
+/****************************************************************************
+ * libs/libc/machine/xtensa/arch_setjmp.S
```

Comment:
Looks like this files sneaked in with a `git add .`
(This looks like what was supposed to be a backup file)